### ISSUE TITLE
unable to see package in cloudpebble

### ISSUE THREAD

**Issue by clach04** [MorrisTimm/pebble-time-machine#1]
Not sure if this is a cloudpebble or time-machine problem, other packages come up in a search in cloudpebble

**Comment by MorrisTimm** [comment]
I don't  know how CloudPebble searches for dependencies.
I haven't done anything different with this package as with my others and they are found.
Maybe it just needs a little more time until it is found in the search.
Until then you can just enter the package name and version manually and it should work just fine.

**Comment by clach04** [comment]
Yes, manually specifying dependency works fine in CloudPebble. Using `pebble-time-machine` and either `1.0.2` or `^1.0.2` works fine for me.

I think Katherine is on vacation so I'll try and ask about CloudPebble search oddities later.
